neqo-http3/src/connection.rs

```diff
@@ -94,6 +97,8 @@ impl Http3Connection {
             streams_have_data_to_send: BTreeSet::new(),
             send_streams: HashMap::new(),
             recv_streams: HashMap::new(),
+            stream_priorities: HashMap::new(),
+            send_priority_update: false,
```

Comment:
I'm not seeing where this is read.  I assume that comes next.

Reply:
I think this should be on the control stream and we actually change priority of a stream only when its update get sent to the peer.

Reply:
Yes, the frame is sent on the control stream, but we can make changes to our sending piece immediately (without waiting for the control stream frames to be sent).

Reply:
That is true.
We need to find a way to handle:
priority was Priority{3, false}
priority_update(0, Priority{4, false}) -> this update has not been sent yet, before the following update
priority_update(0, Priority{3, false}) -> now neqo does not need to send an update anymore.

Reply:
Oh, of course.  It might be easiest to just have a list of pending priority updates as `(u64, Priority)` tuples attached to the control stream.  That would mean a linear search every priority update, but it shouldn't be HUGE numbers of items in that list.  Indeed, you could put a hard limit on the length of the list and throw out old priority updates if the list got too long; we don't want to spend all our uplink on sending priority updates.

Reply:
I'm using VecDeque<u64> now and checking which to Priority to update to from the `recv_conn` variable. I didn't implement anything to throw out some priority updates if there are too many of them.

Reply:
>  Indeed, you could put a hard limit on the length of the list and throw out old priority updates if the list got too long; we don't want to spend all our uplink on sending priority updates.

Do we want to put a limit somewhere or somehow only send priority_updates as long as we don't have something else to send?